kmod_scripts/sof_bootone.sh

```diff
@@ -3,7 +3,7 @@
 dmesg -C
 ./sof_remove.sh
 ./sof_insert.sh
-sleep 1
+sleep 6
```

Comment:
Also not clear why this is needed. 1 second is plenty on all the platforms I have. in what cases do you need 6s?

Reply:
just run: sudo ./sof_bootloop.sh